plugins/templates/create.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const hoek = require('hoek');
+const urlLib = require('url');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/templates',
```

Comment:
This path still seems very contrived. What does it look like to fetch a template? Does the user need to hit an endpoint like `/templates/{id}`? Does the endpoint look like `/templates/<templateName>/<version>`?